packages/mobile/src/components/feed-filter-drawer/FeedFilterDrawer.tsx

```diff
@@ -34,7 +34,7 @@ export const FeedFilterDrawer = () => {
       // Tell the store that the feed is still in view so it can be refetched
       dispatch(feedActions.setInView(true))
       // Force a refresh for at least 10 tiles
-      dispatch(feedActions.refreshInView(true, null, 10))
+      // dispatch(feedActions.refreshInView(true, null, 10))
```

Comment:
delete line instead of comment?

Reply:
great find, this was from a different pr, will undo